### DISCUSSION TITLE
How can I change masthead color??

### DISCUSSION THREAD

**Discussion by hyeok-jong** [mmistakes/minimal-mistakes#3537] [Q&A] [answered]
I read [Issue](https://github.com/mmistakes/minimal-mistakes/issues/573) and I changed `_sass/_navigations.scss`   "greedy nav" .  
Doing so, masthead color changed but whole part of masthead didn't changed.  
It means in below picture, just red part changed

![image](https://user-images.githubusercontent.com/78862026/160290376-29bde876-e547-4d5b-949b-6afdf7c23461.png)

I want change whole part of masthead.

**Comment by iBug** [comment, marked as answer]
You should do it for `.masthead` in `_masthead.scss`.

**Reply by hyeok-jong** [reply]
The rest part of masthead changed!  
Thank you!!

**Comment by JamesTsetsekas** [comment]
@hyeok-jong How did you enable the masthead?! i cant get mine enabled and showing? https://github.com/mmistakes/minimal-mistakes/discussions/3577

**Reply by hyeok-jong** [reply]
In my case I changed in `_navigation.scss`  
```
.greedy-nav {
  position: relative;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: $nav-height;
  background: $masthead-background-color;    <- this part!!!!
```  
However doing so whole color didn't change .  
As @iBug said,  

I add some codes in `_masthead.scss`   

```
.masthead {
  position: relative;
  border-bottom: 1px solid $border-color;
  -webkit-animation: $intro-transition;
  animation: $intro-transition;
  -webkit-animation-delay: 0.15s;
  animation-delay: 0.15s;
  z-index: 20;
  background: $masthead-background-color;   < - here!!!!!!!!!!!!!!!!
```
